extensions/mssql/src/mssql.d.ts

```diff
@@ -259,6 +259,8 @@ declare module 'mssql' {
 		ignoreWorkloadClassifiers: DacDeployOptionPropertyBoolean;
 		ignoreDatabaseWorkloadGroups: DacDeployOptionPropertyBoolean;
 		doNotDropDatabaseWorkloadGroups: DacDeployOptionPropertyBoolean;
+		ignoreSensitivityClassifications: DacDeployOptionPropertyBoolean;
+		optionsMapTable: Map<string, DacDeployOptionPropertyBoolean>;
```

Comment:
do all the different options still need to be passed from STS now that this optionsMapTable is getting passed? Doesn't the optionsMapTable have all the information that's needed for each option: name, value, description?

Reply:
These are not required anymore, as per our discussion way back, we need to work on these as a cleanup story. Do you want me to cover these changes with the current development and add a new PR for that?

Reply:
yes please remove them here and in the STS PR since they are no longer being used to avoid confusion

Reply:
@kisantia Removed the options and their references, Also from STS changes too! Thanks